### PR TITLE
Sbennett/redistls

### PR DESCRIPTION
**Pull request by simone-bennett**
## PR Summary
Changed "older then" to "older than" throughout code base
Created RedisCasheEnterprise Rule, Test
Added Enterprise Cache F1500 to resources.json
Issue # https://github.com/Azure/PSRule.Rules.Azure/issues/1179

## PR Checklist

- [ /] PR has a meaningful title
- [/ ] Summarized changes
- [x] Change is not breaking
- [x] This PR is ready to merge and is not **Work in Progress**
- **Rule changes**
  - [ /] Unit tests created/ updated
  - [ /] Rule documentation created/ updated
  - [/ ] Link to a filed issue
  - [ /] [Change log](https://github.com/Azure/PSRule.Rules.Azure/blob/main/docs/CHANGELOG-v1.md) has been updated with change under unreleased section
- **Other code changes**
  - [ /] Unit tests created/ updated
  - [/ ] Link to a filed issue
  - [/ ] [Change log](https://github.com/Azure/PSRule.Rules.Azure/blob/main/docs/CHANGELOG-v1.md) has been updated with change under unreleased section

### CODE REVIEW

File: docs/en/rules/Azure.RedisEnterprise.MinTLS.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+severity: Critical
+pillar: Security
+category: Data protection
+resource: Azure Cache for Redis Enterprise
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Redis.MinTLS/
```
Comment (jonathanruiz) [inline docs/en/rules/Azure.RedisEnterprise.MinTLS.md:6]: @simone-bennett I believe the issue here with the path of the documentation. It is currently set to `Azure.Redis.MinTLS` when it should be set to `Azure.RedisEnterprise.MinTLS`

File: src/PSRule.Rules.Azure/rules/Azure.RedisEnterprise.Rule.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+#region Rules
+
+
+---
+# Synopsis: Redis Cache Enterprise should reject TLS versions older than 1.2.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.RedisEnterprise.MinTLS
+  ref: AZR-000299
```
Comment (BernieWhite) [inline src/PSRule.Rules.Azure/rules/Azure.RedisEnterprise.Rule.yaml:17]: @simone-bennett Almost there :) Someone took `AZR-000299` try `AZR-000301`.

```suggestion
  ref: AZR-000301
```

**Comment by simone-bennett** [comment]
Requesting to merge in the Redis Enterprise MinTLS rule

**Review by BernieWhite — APPROVED** [review]
@simone-bennett Well done! All good to merge.

**Pull request merged**